packages/vue-app/template/index.js

```diff
@@ -120,7 +120,7 @@ async function createApp (ssrContext) {
         err = err || null
         app.context._errored = Boolean(err)
         err = err ? normalizeError(err) : null
-        const nuxt = this.nuxt || this.$options.nuxt
+        const nuxt = (this && (this.nuxt || (this.$options && this.$options.nuxt))) || app.nuxt
```

Comment:
What do you think expanding to normal if else syntax now and add a comment on top of it pointing to issue why nuxt cannot be available in some cases?

Reply:
Could also be a candidate for an own function ☺️

Reply:
The change allows you to use the new API and does not break backward compatibility.

Reply:
Seems fine for me as a simple work-round but having some docs on top of it regarding reason, next plans (refactor to function), etc could be nice :)

Reply:
What about:

```suggestion
        let nuxt = app.nuxt // to work with @vue/composition-api, see https://github.com/nuxt/nuxt.js/issues/6517#issuecomment-573280207
        if (this) {
          nuxt = this.nuxt || this.$options.nuxt
        }
```